src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -1,16 +1,13 @@
 import { NextFunction, Request, Response } from "express";
```

Comment:
`src/controllers/beneficial.owner.type.controller.ts` is unrelated to this story